filters/private/csf/CSF.cpp

```diff
@@ -0,0 +1,206 @@
+// ======================================================================================
```

Comment:
The couts in this file should be aliased and set via method so we can wire up PDAL's logging.